baseplate/events/queue.py

```diff
@@ -39,24 +39,34 @@ def __init__(self, topic, event_type, timestamp=None, id=None):
         self.id = id or uuid.uuid4()
         self.payload = {}
 
-    def get_field(self, key, obfuscated=False):
+    def get_field(self, key):
         """Get the value of a field in the event.
 
         If the field is not present, :py:data:`None` is returned.
 
         :param str key: The name of the field.
-        :param bool obfuscated: Whether to look for the field in the obfuscated
-            payload.
 
         """
 
-        if not obfuscated:
-            payload = self.payload
-        else:
-            payload = self.payload.get("obfuscated_data", {})
-        return payload.get(key, None)
+        payload = self.payload
+        obfuscated_data = self.payload.get("obfuscated_data", {})
+        interana_excluded = self.payload.get("interana_excluded", {})
+
+        # Check main payload section, the obfuscated section, then
+        # the interana excluded section of the payload before
+        # defaulting to `None`.
+        return payload.get(
```

Comment:
nit: I find this kinda hard to read, especially since `get` is never gonna be able to have more than 2 args, so diff-friendliness isn't super important. The above thing might help a bit, but yeah.